src/test/clojure/clara/test_rules.clj

```diff
@@ -907,6 +907,41 @@
            (set batch-inserts)
            (set single-inserts)))))
 
+(def test-accum-needing-token-partitions-correctly-on-fact-binding
+  (let [q (dsl/parse-query []
+                           [[WindSpeed]
+                            [?ts <- (acc/all) :from [Temperature (= ?loc location)]]])
```

Comment:
I can do that.  The only hesitation I have to adding stuff like that is it can obscure what the test is trying to test.  However, I can see the argument that it is good to get these sort of cases tested for all variations of accumulate nodes.